packages/pg/lib/client.js

```diff
@@ -456,36 +456,12 @@ class Client extends EventEmitter {
     return this._types.getTypeParser(oid, format)
   }
 
-  // Ported from PostgreSQL 9.2.4 source code in src/interfaces/libpq/fe-exec.c
-  escapeIdentifier(str) {
-    return '"' + str.replace(/"/g, '""') + '"'
-  }
-
-  // Ported from PostgreSQL 9.2.4 source code in src/interfaces/libpq/fe-exec.c
-  escapeLiteral(str) {
-    var hasBackslash = false
-    var escaped = "'"
-
-    for (var i = 0; i < str.length; i++) {
-      var c = str[i]
-      if (c === "'") {
-        escaped += c + c
-      } else if (c === '\\') {
-        escaped += c + c
-        hasBackslash = true
-      } else {
-        escaped += c
-      }
-    }
-
-    escaped += "'"
+  // escapeIdentifier and escapeLiteral moved to utility functions & exported
+  // on PG 
+  // re-exported here for backwards compatibility
+  escapeIdentifier = utils.escapeIdentifier
```

Comment:
Need to be defined on the prototype, not as a class field, to avoid breaking backwards compatibility with people who were already using `Client.prototype` directly. Sorry, I should have mentioned that initially.

Reply:
Right, that was one of the workarounds people are using. I'll fix that, and also add test cases that cover the usage of Client.prototype.escape*, as currently they only cover usages of instantiated clients